docs/design/StratisSoftwareDesign.lyx

```diff
@@ -3243,8 +3243,8 @@ parameters of the thinpool device
 \end_layout
 
 \begin_layout Description
-block_devs: an object with 32-byte ASCII hex UUID keys and object values
- with the following structure:
+block_devs: an object with ASCII hex UUID keys and object values with the
+ following structure:
```

Comment:
I'd just be worried with this change it wouldn't be clear the length and format of the uuid we require. Maybe say 32 char?